src/Button/_variables.scss

```diff
@@ -44,34 +46,64 @@ $btn-primary-hover-border-color: var(--pgn-color-btn-hover-border-primary) !defa
 $btn-primary-active-bg: var(--pgn-color-btn-active-bg-primary) !default;
 $btn-primary-active-color: var(--pgn-color-btn-active-text-primary) !default;
 $btn-primary-active-border-color: var(--pgn-color-btn-active-border-primary) !default;
+$btn-primary-focus-focus-outline-color: var(--pgn-color-btn-focus-outline-primary) !default;
```

Comment:
[nit/curious] Is this SCSS variable supposed to have the word `focus` back to back? Is this intended to be `$btn-primary-focus-outline-color`?

Reply:
Thanks
My bad